apps/taskmasters/flowlord/taskmaster.go

```diff
@@ -316,9 +316,12 @@ func (tm *taskMaster) Process(t *task.Task) error {
 		} else { // send to the retry failed topic if retries > p.Retry
 			meta.Set("retry", "failed")
 			t.Meta = meta.Encode()
-			if tm.failedTopic != "-" {
+			if tm.failedTopic != "-" && tm.failedTopic != "" {
 				tm.producer.Send(tm.failedTopic, t.JSONBytes())
 			}
+		}
+
+		if t.Result == task.AlertResult || t.Result == task.ErrResult || t.Result == task.WarnResult {
```

Comment:
As is this will cause every error,alert and warning to be notified on. We need to split it out so only retried errors and alerts are shown messages and warnings are ignored.